kernel/src/hil/sensors.rs

```diff
@@ -36,13 +36,29 @@ pub trait AirQualityDriver<'a> {
     /// Set the client to be notified when the capsule has data ready.
     fn set_client(&self, client: &'a dyn AirQualityClient);
 
+    /// Specify the temperature and humidity used in calculating the air
+    /// quality.
+    ///
+    /// The temperature is specified in degress Celsius and the humidity
+    /// is specified as a percentage.
+    ///
+    /// This is an optional call and doesn't have to be used, but on most
+    /// hardware can be used to improve the measurement accuracy.
+    ///
+    /// This function might return the following errors:
+    /// - `BUSY`: Indicates that the hardware is busy with an existing
+    ///           operation or initialisation/calibration.
+    /// - `NOSUPPORT`: Indicates that this data type isn't supported.
+    fn specify_enviroment(&self, temp: Option<u32>, humidity: Option<u32>)
```

Comment:
Per the #2965 discussion, we should probably use `i32` for temperature here, since it can be less than zero Celsius.